app/services/rootstrap_member_repositories_discriminator.rb

```diff
@@ -0,0 +1,15 @@
+class RootstrapMemberRepositoriesDiscriminator < BaseService
+  def initialize(repositories)
+    @repositories = repositories
+  end
+
+  def call
+    @repositories.reject { |repository| rs_members.include? repository[:owner][:login] }
+  end
+
+  private
+
+  def rs_members
```

Comment:
not sure if we want to make this repo "abstract" of RS or not. cc @santiagovidal

Reply:
If possible, yes. In this case, we can name it `company_members` instead of `rs_members` for example.